faqs.rst

```diff
@@ -50,24 +50,38 @@ or `sched <https://github.com/flux-framework/flux-sched/issues>`_.
 
 .. _not_managing_all_resources:
 
-Why is Flux not discovering and managing all of the resources on the system/node?
-=================================================================================
+Why is Flux ignoring my Nvidia GPUs?
+====================================
 
-This can be due to various bind flags that need to be passed parallel launcher
-that started Flux. For example at LLNL you must pass ``--mpibind=off`` to
-``srun`` and ``--bind=none`` to ``jsrun``.
```

Comment:
I think there was indeed an issue where default mpibind and possibly `jsrun` binding made it so that hwloc couldn't detect GPUs (maybe they set `CUDA_VISIBLE_DEVICES` to something incorrect by default?)

If we haven't verified that these options are no longer needed, perhaps we leave this in as something to try, e.g.

> If running under Slurm or LSF, try disabling any CPU affinity options, e.g. `--mpibind=off` with Slurm or `--bind=none` with `jsrun`.

Reply:
Actually, now that I think about this more, I'm not sure the binding options influenced discovery of GPUs. If you'd prefer, leave that out and we can add it back in if it deemed to be necessary in the future.

Reply:
> Actually, now that I think about this more, I'm not sure the binding options influenced discovery of GPUs. If you'd prefer, leave that out and we can add it back in if it deemed to be necessary in the future.

Should we contrive another question about cpu affinity that would let us keep those options documented in the FAQ and possibly show how the Flux cpu affinity options work and how to verify what's really going on with your app?  I'm not sure I have a great question in mind, so just throwing that out there.

For this PR I'll go ahead and leave that stuff out.

Reply:
Probably, since that seemed to be the intent of this faq entry before

Reply:
I guess the point to get across is that when flux discovers resources via hwloc, it honors the current core and gpu bindings, so if resources are missing, affinity and binding from the parent RM should be checked

Reply:
Gotcha.  Just pushed a new question based on your comments.  It's close to the original question so I squashed the two commits into one that says it is splitting the original question (and renamed this PR).  Want to have a look and see if this works?

Reply:
Yeah, your edits were already much clearer, but the new entry also handles what I think was implicitly answered before. Thanks!